### PR TITLE
Add Event lookup

### PR DESCRIPTION
**Pull request by Haschikeks**
Add new Lookup Function for [Event](https://musicbrainz.org/doc/Event)

This PR is connected to #3

### CODE REVIEW

File: src/musicbrainz-api.ts [hunk 1]
```diff
@@ -363,6 +363,15 @@ export class MusicBrainzApi {
     return this.lookupEntity<mb.IUrl, UrlIncludes>('url', urlId, inc);
   }
 
+  /**
+   * Lookup Event
+   * @param eventId Event MBID
+   * @param inc EventIncludes
```
Comment (Borewit) [inline src/musicbrainz-api.ts:369]: I suggest clear parameter names
```suggestion
   * @param eventIncludes EventIncludes
```

Comment (Borewit) [inline src/musicbrainz-api.ts:369]:
```suggestion
   * @param inc List of sub-queries to enable
```

**Review by Borewit — CHANGES_REQUESTED** [review]

**Review by Haschikeks — COMMENTED** [review]
Updated parameter names

**Review by Borewit — APPROVED** [review]

**Comment by Borewit** [comment]
We need to describe new functionality in the README, realized just after merging

**Pull request merged**